### PR TITLE
updated sample to use the 2.0 liberty plugin

### PR DESCRIPTION
**Pull request by mattbsox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ NOTE: When running this Java EE 7 sample using eclipse and WDT, some validation
 
 ## Running with Maven
 
-This project can be build with [Apache Maven](http://maven.apache.org/). The project uses [Liberty Maven Plug-in][] to automatically download and install Liberty profile runtime from the [Liberty repository](https://developer.ibm.com/wasdev/downloads/). Liberty Maven Plug-in is also used to create, configure, and run the application on the Liberty server. 
+This project can be built with [Apache Maven](http://maven.apache.org/). The project uses [Liberty Maven Plug-in](https://github.com/WASdev/ci.maven) to automatically download and install Liberty profile runtime from the [Liberty repository](https://developer.ibm.com/wasdev/downloads/). Liberty Maven Plug-in is also used to create, configure, and run the application on the Liberty server.
```
Comment (patricktiu) [inline README.md]: nstruction for **Running with Maven running** in README.md 

This project can be built with Apache Maven. The project uses [Liberty Maven Plug-in](https://github.com/WASdev/ci.maven) to automatically download and install Liberty ~~profile~~ `with Java EE7 Full Platform`  runtime from the ~~Liberty repository~~ `Maven Central`. Liberty Maven Plug-in is also used to create, configure, and run the application on the Liberty server.

Comment (patricktiu) [inline README.md]: We need to update the instruction of **Running in Eclipse**.  


1.    Download and install Eclipse with the WebSphere Developer Tools.
2.   ~~Create a new Liberty Profile Server.~~
3.    Clone this repository.
4.    Import the sample into Eclipse using File -> Import -> Maven -> Existing Maven Projects option.
5.    ~~Edit the server.xml file located in the Servers -> DefaultServer folder in EnterpriseExporer. Copy the server.xml features from this project, found under the <featureManager> in /src/main/wlp/server.xml~~
6.   ~~Deploy the sample into Liberty server.~~ Right click on the project and select Run As -> Run on Server option. Find and select the Liberty profile server and press Finish.
7.    Go to: http://localhost:9080/sample.javaee7.jms/

File: src/main/wlp/server.xml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 <server description="new server">
 
     <!-- Enable features -->
-<featureManager>
-       <feature>jms-2.0</feature>
```
Comment (patricktiu) [inline src/main/wlp/server.xml]: Did you accidentally remove the jms-2.0 feature?

File: pom.xml [hunk 3]
```diff
@@ -37,42 +37,67 @@
...
+        <dependency>
+            <groupId>javax.jms</groupId>
+            <artifactId>javax.jms-api</artifactId>
+            <scope>provided</scope>
+            <version>2.0</version>
         </dependency>
     </dependencies>
 
     <build>
-        <finalName>${project.artifactId}</finalName>
```
Comment (patricktiu) [inline pom.xml]: Other samples are using `<finalName>`.  Let's continue to use it.

File: pom.xml [hunk 4]
```diff
@@ -37,42 +37,67 @@
...
         <plugins>
             <plugin>
                 <groupId>net.wasdev.wlp.maven.plugins</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
+                <version>2.0-SNAPSHOT</version>
                 <configuration>
                     <configFile>src/main/wlp/server.xml</configFile>
+                    <installAppPackages>project</installAppPackages>
+                    <looseApplication>true</looseApplication>
+                    <stripVersion>true</stripVersion>
```
Comment (patricktiu) [inline pom.xml]: You can delete line 58. The `<stripVersion>` is not in the liberty-maven-app-parent pom

File: pom.xml [hunk 5]
```diff
@@ -37,42 +37,67 @@
...
+                    <assemblyArtifact>
+                        <groupId>com.ibm.websphere.appserver.runtime</groupId>
+                        <artifactId>wlp-javaee7</artifactId>
+                        <version>17.0.0.1</version>
+                        <type>zip</type>
+                    </assemblyArtifact>
                     <bootstrapProperties>
-                        <appLocation>../../../../../${project.build.finalName}</appLocation>
+                        <appLocation>${project.artifactId}.war</appLocation>
```
Comment (patricktiu) [inline pom.xml]: change to `<appLocation>${project.build.finalName}.war</appLocation>`

**Review by patricktiu — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 5 above)
Comment (mattbsox) [reply pom.xml]: Should I go change this in the rest of the samples as well?

File: src/main/wlp/server.xml (hunk 2 above)
Comment (mattbsox) [reply src/main/wlp/server.xml]: Ya, that was an accident. I'll add it back.

Comment (mattbsox) [reply src/main/wlp/server.xml]: Do the wasJmsClient-2.0 and wasJmsServer-1.0 take care of the jms-2.0 feature? I think that's why I took it out in the first place. https://www.ibm.com/support/knowledgecenter/SSAW57_liberty/com.ibm.websphere.wlp.nd.multiplatform.doc/ae/twlp_msg.html

Comment (mattbsox) [reply src/main/wlp/server.xml]: We determined that wasJmsServer and wasJmsClient both pull in the jms-2.0 feature when they are installed, so it doesn't need to be set as a feature.

**Review by patricktiu — APPROVED** [review]

**Review by patricktiu — APPROVED** [review]

**Pull request merged**